beancount-tests.el

```diff
@@ -1,6 +1,7 @@
 ;;; beancount-test.el --- ERT for beancount-mode -*- lexical-binding: t -*-
 
 ;; Copyright 2019 Daniele Nicolodi <daniele@grinta.net>
+;; Copyright 2019 Alex Smith <aes7mv@virginia.edu>
```

Comment:
2019?

Reply:
Whoops, that was a case of "thinking one thing but typing what I'm looking at". Will fix.